docs/sections/Docs-Forms/index.jsx

```diff
@@ -72,13 +72,21 @@ export default class Forms extends React.Component {
 
         <section className="doc-bottom-space-large">
           <h3 className="doc-heading">Form Error</h3>
-          <p>To apply, .message</p>
+          <p>To apply to bottom, .message</p>
```

Comment:
Indicate it as default rather 
or like this
`To apply, .message (position in bottom by default)
or
not indicate the `bottom`